scss/components/_fonts.scss

```diff
@@ -0,0 +1,23 @@
+//NexaRegular
```

Comment:
fonts does not look like a component - you could put this file directly in `scss` folder

Reply:
Done